src/main/kotlin/no/nav/familie/ef/sak/opplysninger/mapper/BarnMedSamværMapper.kt

```diff
@@ -168,17 +169,15 @@ object BarnMedSamværMapper {
             bosattINorge = pdlAnnenForelder.bostedsadresse.gjeldende()?.utenlandskAdresse?.let { false } ?: true,
             land = pdlAnnenForelder.bostedsadresse.gjeldende()?.utenlandskAdresse?.landkode,
             tidligereVedtaksperioder = pdlAnnenForelder.tidligereVedtaksperioder?.tilDto(),
-            langAvstandTilSøker = langAvstandTilSøker(søkerAdresse, pdlAnnenForelder.bostedsadresse.gjeldende())
+            avstandTilSøker = langAvstandTilSøker(søkerAdresse, pdlAnnenForelder.bostedsadresse.gjeldende())
         )
     }
     private fun langAvstandTilSøker(
         søkerAdresse: List<Bostedsadresse>,
         bostedsadresse: Bostedsadresse?
-    ) =
+    ): AvstandTilSøkerDto =
         bostedsadresse
             ?.vegadresse
             ?.fjerneBoforhold(søkerAdresse.gjeldende()?.vegadresse)
-            ?.takeIf { it }
-            ?.let { LangAvstandTilSøker.JA }
-            ?: LangAvstandTilSøker.UKJENT
+            ?: AvstandTilSøkerDto(avstandIKm = null, langAvstandTilSøker = LangAvstandTilSøker.UKJENT)
```

Comment:
Vet ikke helt om jeg skjønte koden fra før, 
Det virket som at den alltid ga JA tidligere, nå gir den alltid ukjent?

Reply:
Den ga "JA" hvis `it` var true. Nå gir den tilbake `it` hvis den finnes - ellers `UKJENT`